applications/GeoMechanicsApplication/custom_constitutive/coulomb_with_tension_cut_off_impl.cpp

```diff
@@ -24,6 +24,18 @@ namespace
 
 using namespace Kratos;
 
+double CalculatePlasticMultiplier(const Vector& rSigmaTau,
+                                  const Vector& rDerivativeOfFlowFunction,
+                                  double        FrictionAngleInRadians,
+                                  double        Cohesion)
+{
+    const auto sin_phi   = std::sin(FrictionAngleInRadians);
+    const auto numerator = sin_phi * rDerivativeOfFlowFunction[0] + rDerivativeOfFlowFunction[1];
+    const auto lambda =
+        (Cohesion * std::cos(FrictionAngleInRadians) - rSigmaTau[0] * sin_phi - rSigmaTau[1]) / numerator;
+    return lambda;
```

Comment:
This could be simplified a little bit, but it's more of a personal preference, so I leave it up to you:
 ```suggestion
    return (Cohesion * std::cos(FrictionAngleInRadians) - rSigmaTau[0] * sin_phi - rSigmaTau[1]) / numerator;
```

Reply:
I woul like to link this to the variable lambda in the mathematical formulations. I think mentioning lamba makes it more clear, what this function is meant for.